pegasus/forms/pd_workshop_survey.rb

```diff
@@ -76,6 +76,14 @@ def self.normalize(data)
     result
   end
 
+  def self.process_(form)
+    # Save this form id in the relevant dashboard pd_enrollment row
+    id = form[:id]
+    data = JSON.load(row[:data])
+    enrollment_id = data['enrollment_id_i']
+    DASHBOARD_DB[:pd_enrollments][enrollment_id].update(survey_id: id)
```

Comment:
Yes. It's the sequel equivalent of ActiveRecord's `find_by_id`. See http://sequel.jeremyevans.net/rdoc/files/doc/querying_rdoc.html#label-Retrieving+a+Single+Object

But I realized I did it wrong. It's supposed to be `DASHBOARD_DB[:pd_enrollments][id: enrollment_id]...` or `DASHBOARD_DB[:pd_enrollments].where(id: enrollment_id).first`